tests/rados/test_track_availibility_status.py

```diff
@@ -0,0 +1,604 @@
+                f"parameter is set to false"
+            )
+            log.error(msg_error)
+            return 1
+        msg_info = (
+            f"The {availability_msg} is displayed in the output  when the enable_availability_tracking "
+            f"parameter is set to false"
+        )
+        log.info(msg_info)
+        mon_obj.set_config(
+            section="mon", name="enable_availability_tracking", value="true"
+        )
+        result_all_tests = test_all_values(client, pool_name)
+        if not result_all_tests:
+            msg_error = f"  Case 8: The track availability tests are failed on {pool_name} during balancer activity"
+            log.error(msg_error)
+            return 1
+        log.info(
+            "=== Case 11: Verification enable_availability_tracking parameter completed  ==="
+        )
+        log.info(
+            "=== Case 12: Verification clear-availability-status command on pool  ==="
+        )
+
+        cmd_clear_pool_status = f"ceph osd pool clear-availability-status {pool_name}"
+        client.exec_command(cmd=cmd_clear_pool_status)
+        out_put = get_pool_availability_status(client)
+        selected_pool_details = out_put[pool_name]
+        if (
+            int(selected_pool_details[2]) != 0
+            and int(selected_pool_details[4]) != 1
+            and int(selected_pool_details[5]) != 1
+        ):
+            msg_error = f"The clear_availability_status command  not clear the {pool_name} pool data"
+            log.error(msg_error)
+            return 1
+        msg_info = (
+            f"The clear_availability_status command clear the {pool_name} pool data"
+        )
+        log.info(msg_info)
+        log.info(
+            "=== Case 12: Verification clear-availability-status command on pool completed  ==="
+        )
+
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
+    finally:
+        log.info(
+            "\n \n ************** Execution of finally block begins here *************** \n \n"
+        )
+        # if config.get("delete_pool"):
+        #     method_should_succeed(rados_object.delete_pool, pool_name)
+        mon_obj.set_config(
+            section="mon", name="enable_availability_tracking", value="true"
+        )
+    return 0
+
+
+# ceph osd pool availability-status
+
+
+def get_pool_availability_status(client):
+    """
+    Method is used to get the availability status of cluster
+    Args:
+        client : client object
+    Returns:
+        available_data_in_dict :Available data in dictionary.
+        For example:
+              POOL             UPTIME  DOWNTIME    NUMFAILURES  MTBF  MTTR  SCORE  AVAILABLE
+              .mgr                 4d     0s            0        0s    0s      1          1
+          Output : [.mgr] ->['4d','0s','0','0s','0s','1','1']
+    """
+    cmd = "ceph osd pool availability-status"
+    cmd_out_put = ""
```

Comment:
no need to initialize a empty string. the cmd o/p below would be tuple

Reply:
Removed the line.